### ISSUE TITLE
How to set common Footer. 

### ISSUE THREAD

**Issue by aarunraja** [stoplightio/desktop#364]
Hi..,
Is there a way to set common Footer in all my pages (oas2, hub).  to display contains elements that include a company's logo, website navigation, and contact information.

**Comment by aarunraja** [comment]
@rossmcdonald thanks for the reply. Currently i have only those as requirement.  
can i get timeline. so that i can estimate in my Backlog.

**Comment by rsrinivasanhome** [comment]
@rossmcdonald  is there any work around available today ?

**Comment by rossmcdonald** [comment]
@rsrinivasanhome There is no way to set a site-wide footer at this time, however you can create a custom footer HTML file and reference that at the end of every page in your hub to emulate a footer.

**Comment by rossmcdonald** [comment]
@aarunraja This item has been added to our feature backlog for review by our engineering team. I don't have a timeline that I can share at this time, but I'll let you know as soon as they've been able to review and it's added to a sprint.

**Comment by aarunraja** [comment]
@rossmcdonald Thanks.. I can able to achieve footer for a Hub file by added HTML tag at bottom. but Most of my pages are OpenAPI page type. is there a way to add HTML tag in  OpenAPI page type.

**Comment by rossmcdonald** [comment]
@aarunraja No, currently you can't customize the auto-generated swagger output. I'll let you know as soon as there's an update from the team.

**Comment by rsrinivasanhome** [comment]
@rossmcdonald  do we have any idea now , when the feature would be available  ?
